src/pyOpenMS/pxds/FIAMSDataProcessor.pxd

```diff
@@ -16,14 +16,52 @@ cdef extern from "<OpenMS/ANALYSIS/ID/FIAMSDataProcessor.h>" namespace "OpenMS":
         #
         # wrap-inherits:
         #  DefaultParamHandler
-        FIAMSDataProcessor() nogil except +
-        FIAMSDataProcessor(FIAMSDataProcessor) nogil except +
+        FIAMSDataProcessor() nogil except + # wrap-doc:Data processing for FIA-MS data
+        FIAMSDataProcessor(FIAMSDataProcessor) nogil except + # wrap-ignore
```

Comment:
IMHO `# wrap-ignore `should not be needed here.

@hroest When is a # wrap-ignore needed for the copy constructor? Or is there no specific rule?

Reply:
@oliveralka should I remove it?

Reply:
As far as I have seen the  `# wrap-ignore` for the copy constructor is also automatically added in the `PythonExtensionChecker.py`. So IMHO, let's leave the # wrap-ignore for now. Any objections @timosachsenberg @hroest